### ISSUE TITLE
Message when running the script file

### ISSUE THREAD

**Issue by rdstern** [africanmathsinitiative/R-Instat#4216]
I have been trying the script file and am very happy - so far.

My main triumph has been to use it within R-Instat to produce a boxplot "by year", but with the Year column numeric - we can't do that from the dialogue.

I give the example as a separate issue, for information.  But I found the warning message - every time - got annoying.  Could we have a checkbox (or button, or ...) within the warning - to turn it off within a session.  I am happy that it starts that way each time we invoke R-Instat, but not forced on us each time we are so naughty as to use the feature.  (Is that a David and Danny hangover?)

**Comment by maxwellfundi** [comment]
The issue on a has been addressed already. For the issue b, we currently cant not run the script window while dialog is open because they are the topmost forms and so they have to be hidden to be able to click on other controls on frmMain. Whether we can change that, I am not sure maybe @dannyparsons  and @volloholic  can advice on that. 

It would be good to have the message box with the "Dont Show this again in current session" but this (in my current understanding) would call to change the current message box to a form so that we can easily do that.

**Comment by dannyparsons** [comment]
Yes that's right it would need to be a form, I think. If we did that I think it would be nice to have the code that is about to be run display in a text box in the form as well. Not urgent to do these though.

I would have it not just for the current session but "forever" and then it is just like other options in the Options dialog.

**Comment by lloyddewit** [comment]
I understand that the remaining parts of this issue are:
- In the script window, do not display a warning each time the user selects 'Run All' (maybe have a checkbox 'do not show this again' or something similar)
- Allow the script window to be used even if a dialog is open

@rdstern If I misunderstood, then please let me know, thanks
